akka-remote/src/test/scala/akka/remote/artery/tcp/ssl/RotatingKeysSSLEngineProviderSpec.scala

```diff
@@ -271,11 +270,9 @@ abstract class RotatingKeysSSLEngineProviderSpec(extraConfig: String)
   override def beforeTermination(): Unit = {
     systemsToTerminate.foreach { systemToTerminate =>
       system.log.info(s"Terminating $systemToTerminate...")
-      val now = System.nanoTime()
-      Await.result(systemToTerminate.terminate(), 8.seconds.dilated)
-      val lasted = System.nanoTime() - now
-      system.log.info(s"Terminated $systemToTerminate after ${lasted / 1000000} ms")
+      systemToTerminate.terminate()
     }
+    systemsToTerminate.foreach(shutdown(_, verifySystemShutdown = true))
```

Comment:
a bit weird to do *both* `terminate` and `shutdown`, but I guess this is fine since `terminate` is idempotent and this makes it happen in parallel.

Reply:
I want to start all the terminations in parallel and then use `shutdown` to capture extra information from the failure (when there's one).